bindata/network/multus/004-configmap.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cni-copy-bash-script
+  namespace: openshift-multus
+  labels:
+    app: openshift-multus
+data:
+  copycni.sh: |-
+    #!/bin/bash
+    set -x
+    # source our os-release variables.
+    if [ -f "/host/etc-os-release/os-release" ]; then
+      echo "Sourced variables from /host/etc-os-release/os-release"
+      . /host/etc-os-release/os-release
+    else
+      echo "Sourced variables from /host/usr/lib/os-release"
+      . /host/usr/lib/os-release
+    fi
+    rhelmajor=
+    # detect which version we're using in order to copy the proper binaries
+    case "${ID}" in
+      rhcos) rhelmajor=8
+      ;;
+      rhel) rhelmajor=$(echo "${VERSION_ID}" | cut -f 1 -d .) 
+      ;;
+      *) echo "WARNING: Unsupported OS ID=${ID}" 
```

Comment:
This should probably be a fatal error

Reply:
Good call updated the echo output and added `exit 1`